mod/node-api/handlers/beacon/routes.go

```diff
@@ -153,6 +153,11 @@ func (h *Handler[_, ContextT, _, _]) RegisterRoutes(
 			Path:    "/eth/v1/beacon/deposit_snapshot",
```

Comment:
is there a way to modularize slug / url building?

Reply:
One enhancement that I could of is to have base path set for the module and then let these paths be post that - something like "/deposit_snapshot`" in above case.

How about having a URL builder struct for constructing the URLs, not sure if that would much helpful as it would be similar to this. Will think about other approaches.